gems/pending/spec/middleware/events/middleware_event_monitor_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "middleware/events/middleware_event_monitor"
```

Comment:
when moved to `ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Monitor` this require can also go away imho